src/vuex/modules/overlay/index.js

```diff
@@ -37,6 +37,7 @@ const mutations = {
     [UNMOUNT_OVERLAY](state, { id }) {
         Vue.delete(state.overlays, id);
     },
+    // TODO: Only use the part of the payload that is being requested
```

Comment:
please avoid TODO: comments, we can use asana o github issues to keep track of the missing feature or improvements